libs/coin-modules/coin-sui/src/logic/index.ts

```diff
@@ -4,4 +4,5 @@ export { broadcast } from "./broadcast";
 export { combine } from "./combine";
 export { getBalance } from "./getBalance";
 export { lastBlock } from "./lastBlock";
+export { getBlock, getBlockInfo } from "./getBlock";
```

Comment:
A little nitpicking, but we could regroup all "block" functions (`lastBlock`, `getblock` and `getBlockInfo`) into one file in the logic directory. Let say: `block.ts`

Reply:
Yes sure, I'll do it in separate PR though, there are many concurrent PRs on SUI coin module right now, conflict resolution is difficult

Reply:
To be done in followup PR